packages/core/src/lib/select/searchable-select.svelte

```diff
@@ -175,9 +178,15 @@ $: activeElement = activeOption
   ? optionElements[activeOption.option.value]
   : undefined;
 
-$: if (typeof activeElement?.scrollIntoView === 'function') {
-  activeElement.scrollIntoView({ block: 'nearest' });
-}
+const scrollIntoView = (element: HTMLElement | undefined): void => {
+  if (typeof element?.scrollIntoView === 'function') {
+    onNextFrame(() => {
+      element.scrollIntoView({ block: 'nearest', inline: 'nearest' });
+    });
```

Comment:
[opt] Without reviewing this PR, It would not be intuitive to me why this is required. If you agree that this is not obviously intuitive, a short comment would be good to have.